[2023-03-15T17:18:11.053+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: 01_dag_test.read_from_db manual__2023-03-15T17:18:08.375134+00:00 [queued]>
[2023-03-15T17:18:11.059+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: 01_dag_test.read_from_db manual__2023-03-15T17:18:08.375134+00:00 [queued]>
[2023-03-15T17:18:11.060+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-03-15T17:18:11.060+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-03-15T17:18:11.061+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-03-15T17:18:11.069+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): read_from_db> on 2023-03-15 17:18:08.375134+00:00
[2023-03-15T17:18:11.074+0000] {standard_task_runner.py:55} INFO - Started process 5035 to run task
[2023-03-15T17:18:11.078+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', '01_dag_test', 'read_from_db', 'manual__2023-03-15T17:18:08.375134+00:00', '--job-id', '1141', '--raw', '--subdir', 'DAGS_FOLDER/dag-test.py', '--cfg-path', '/tmp/tmpd2ebmt7p']
[2023-03-15T17:18:11.080+0000] {standard_task_runner.py:83} INFO - Job 1141: Subtask read_from_db
[2023-03-15T17:18:11.146+0000] {task_command.py:376} INFO - Running <TaskInstance: 01_dag_test.read_from_db manual__2023-03-15T17:18:08.375134+00:00 [running]> on host f11e2626d663
[2023-03-15T17:18:11.198+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=01_dag_test
AIRFLOW_CTX_TASK_ID=read_from_db
AIRFLOW_CTX_EXECUTION_DATE=2023-03-15T17:18:08.375134+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-15T17:18:08.375134+00:00
[2023-03-15T17:18:11.209+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-03-15T17:18:11.216+0000] {sql.py:315} INFO - Running statement: SELECT * FROM pg_catalog.pg_tables;, parameters: None
[2023-03-15T17:18:11.219+0000] {sql.py:324} INFO - Rows affected: 108
[2023-03-15T17:18:11.230+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=01_dag_test, task_id=read_from_db, execution_date=20230315T171808, start_date=20230315T171811, end_date=20230315T171811
[2023-03-15T17:18:11.253+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-03-15T17:18:11.268+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
